src/permissionGroups/components/PermissionGroupDeleteDialog/PermissionGroupDeleteDialog.tsx

```diff
@@ -4,17 +4,20 @@ import { FormattedMessage, useIntl } from "react-intl";
 
 import ActionDialog from "@saleor/components/ActionDialog";
 import { ConfirmButtonTransitionState } from "@saleor/components/ConfirmButton";
+import { Typography } from "@material-ui/core";
 
 export interface PermissionDeleteDialogProps {
   confirmButtonState: ConfirmButtonTransitionState;
-  open: boolean;
+  error?: string;
```

Comment:
Wouldn't it be better if we format message in this component instead of doing it on a logic level?
This way we wouldn't mix logic and view layers.